build.override.targets

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="WcfSetup" Condition="'$(WithCategories)' == 'OuterLoop'" BeforeTargets="Test">
+    <Exec Command="$(MySetupScript) > $(MyLogFile)" />
+    <ReadLinesFromFile File="$(MyLogFile)" >
+      <Output TaskParameter="Lines" PropertyName="ReturnCode"/>
+    </ReadLinesFromFile>
+    <!-- This PropertyGroup may not be needed for the WcfCleanup Target to access ReturnCode -->
+    <PropertyGroup>
+      <SetupReturnCode>$(ReturnCode)</SetupReturnCode>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="WcfCleanup" Condition="'$(WithCategories)' == 'OuterLoop'" AfterTargets="Test">
```

Comment:
We'll need a smarter test to know if we are running OuterLoop for these 2 reasons:
1. WithCategories can be a semi-colon separated list
2. You can choose OuterLoop with /p:OuterLoop=true (I recently learned this)
3. You can trigger an OuterLoop by setting xunit traits directly on the command line.
We'll need to investigate when and how buildtools converts $(WithCategories) and $(OuterLoop) into xunit trait arguments to be accurate. But for now, I think we can get by with tests that look like this (my syntax may be off):
```
Condition="$(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true'"
```